src/ontology/wbphenotype-edit.owl

```diff
@@ -2626,6 +2627,14 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://pu
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/WBPhenotype_0000002> "kinker"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/WBPhenotype_0000002> <http://purl.obolibrary.org/obo/WBPhenotype_0004017>)
 
+# Class: <http://purl.obolibrary.org/obo/WBPhenotype_0000003> (<http://purl.obolibrary.org/obo/WBPhenotype_0000003>)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/WBPhenotype_0000003> "obsolete"^^xsd:string)
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0100001> <http://purl.obolibrary.org/obo/WBPhenotype_0000003> <http://purl.obolibrary.org/obo/WBPhenotype_0001309>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/WBPhenotype_0000003> "C_elegans_phenotype_ontology"^^xsd:string)
```

Comment:
Is this necessary for WormBase somehow? Why do we need the obonamespace?

Reply:
Again, just following what is already the case for existing terms, including obsoleted terms. If we decide we don't need the "has_obo_namespace" annotation I can omit it